bookkeeper-server/src/main/java/org/apache/bookkeeper/util/LocalBookKeeper.java

```diff
@@ -299,6 +297,9 @@ static void startLocalBookiesInternal(ServerConfiguration conf,
                 }
                 throw ie;
             }
+        } catch (Exception e) {
```

Comment:
If we catch the exception here, it means we are not exiting the process. I would say to not catch here.